### DISCUSSION TITLE
How to get a BAA

### DISCUSSION THREAD

**Discussion by eltoob** [deepgram/community#760] [General help] [answered]
Hi,
We are currently looking to test out deepgram with our AI medical scribe and I would love to see how we can get a BAA with deepgram.
Cheers,
Elie

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment]
@eltoob let me pass this request to our Sales Team to see if they can help you out here.

**Reply by jpvajda** [reply, marked as answer]
@eltoob I confirmed that you can reach out to [Caitlin Conscolluela](mailto:caitlin.coscolluela@deepgram.com) for more information on a BAA.
